app/views/physical_ticket/show.pdf.prawn

```diff
@@ -0,0 +1,64 @@
+prawn_document(filename: @pdf_filename, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
```

Comment:
Sorry my bad, it should be `@file_name`. But yes it was working, Pdf was just opening in the browser instead of getting downloaded as there was no file name.